deps/jemalloc/include/jemalloc/jemalloc.sh

```diff
@@ -5,6 +5,11 @@ objroot=$1
 cat <<EOF
 #ifndef JEMALLOC_H_
 #define JEMALLOC_H_
+
+/* A macro that is used to indicate that this the jemalloc vendored with the project
+ * and has been tested with active defragmentation. */
+#define VALKEY_VENDORED_JEMALLOC yes
```

Comment:
Nit: If some code does `#if VALKEY_VENDORED_JEMALLOC`, it expands to invalid C code (or to a variable named yes). It's better that it expands to 1.

```suggestion
#define VALKEY_VENDORED_JEMALLOC 1
```